downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc

```diff
@@ -1,34 +1,34 @@
-
+// Module included in the following assemblies:
+// assembly-disconnected-installation.adoc
 
 [id="con-aap-installation-on-disconnected-rhel_{context}"]
 
-
-= {PlatformNameShort} installation on disconnected RHEL
+= Installing {PlatformNameShort} on disconnected RHEL
```

Comment:
I think we've mixed up conceptual and procedural modules here. Based on the name of this file, this should be conceptual information and the title should reflect that (e.g. Disconnected installation, like the title of the chapter).

Reply:
changed it back to the previous title